css-color/Overview.bs

```diff
@@ -60,12 +60,11 @@ Foreground Color: the 'color' property</h2>
 	Inherited: yes
 	Percentages: N/A
 	Media: visual
-	Computed value: an RGBA color
+	Computed value: See <a>resolving color values</a>
```

Comment:
I think this is actually wrong for the <code>color</code> property, since for <code>color</code>, <code>currentColor</code> should be resolved at computed value time.  (I can't think of how this is observable today, but I'd much prefer it be defined this way.)